src/django/api/views/facility/facility_list_view_set.py

```diff
@@ -206,176 +146,27 @@ def create(self, request):
                     f'FacilityList {replaces.pk} has already been replaced.'
                 )
 
-        header, rows = self._extract_header_rows(csv_file, request)
-
-        new_list = FacilityList(
+        new_list = FacilityList.objects.create(
             name=name,
             description=description,
-            file_name=csv_file.name,
-            file=csv_file,
-            header=header,
```

Comment:
It will be filled in this file on row 49 -> https://github.com/opensupplyhub/open-supply-hub/pull/356/files#diff-4f818073994776a52735b5c329922f9ae8fd550f06d9529e79229abc1396c8b9R49
It will happen after receiving data from ContriCleaner.

Reply:
I see, thank you!